### DISCUSSION TITLE
everything works fine except the extended view distance

### DISCUSSION THREAD

**Discussion by rohandoge** [mircokroon/minecraft-world-downloader#250] [Q&A]
I can connect fine to a server, I see the minimap getting updated in real time and the world is even downloaded correctly (I opened the world file and the chunks where there) yet when I'm playing the render is still capped at the server's maximum even when mkoving the slider. 

Is there an extra  step I should do to make it work?
thanks in advance

**Comment by mircokroon** [comment]
Some servers misreport their render distance, so you may have to turn on "measure render distance" to ignore that

**Reply by rohandoge** [reply]
I tried doing that but doesnt seem to work.

Here is a vid of me trying so maybe it can help https://www.youtube.com/watch?v=ysY7r86X3yQ

**Reply by mircokroon** [reply]
Hmm it does seem to be loading the chunks, just for some reason the client isn't rendering them. Can you try maxing out the in-game view distance, and perhaps running Minecraft with the log open (is an option somewhere in the launcher) to see if there's any issues with loading the chunks

**Reply by rohandoge** [reply]
hope this helps https://www.youtube.com/watch?v=vgRsN-HCStY
